plugins/autocertkit

```diff
@@ -609,6 +609,63 @@ def deploy_iperf(session, args):
     return json_dumps("OK")
 
 
+@log_exceptions
+def deploy_vf_driver(session, args):
+    """Install VF driver on a specified VM"""
+    log.debug("deploy VF driver plugin call")
+    vm_ref = validate_exists(args, 'vm_ref')
+    username = validate_exists(args, 'username')
+    password = validate_exists(args, 'password')
+    package  = args['package']
+    driver_name = args['driver_name']
+
+    if not session.xenapi.VM.get_is_control_domain(vm_ref):
+        i = 0
+        limit = 10
+        while i < limit:
+            try:
+                install_ssh_key(session, vm_ref, username, password)
```

Comment:
I believe we already have to do this in the plugin for other remote calls. Can we refactor to use the same mechanism for key exchange?

Reply:
Yes I think, I saw lot of place call install_ssh_key(), I am to able to refactor it later because at present I have to let the feature ready firstly before deadline :)